node/service/src/chain_spec.rs

```diff
@@ -447,8 +455,14 @@ pub fn darwinia_build_spec_genesis() -> DarwiniaGenesisConfig {
 	)
 	.unwrap()
 	{
-		if kton > KTON_EXISTENTIAL_DEPOSIT {
-			ktons.push((fixed_hex_bytes_unchecked!(address, 32).into(), kton));
+		if kton >= KTON_EXISTENTIAL_DEPOSIT {
+			let account_id: AccountId32 = fixed_hex_bytes_unchecked!(address, 32).into();
+
+			ktons.push((account_id.clone(), kton));
```

Comment:
KTON could also have duplicate possibilities, for example, different account from Ethereum and Tron could transfer to same darwinia address.

Or make sure ```ethereum-tron-genesis-swap-kton.json``` did the right merge and add.

@freehere107